view/theme/frio/templates/theme_settings.tpl

```diff
@@ -6,6 +6,7 @@
 
 {{include file="field_select.tpl" field=$scheme}}
 
+{{if $nav_bg}}{{include file="field_input.tpl" field=$share_string}}{{/if}}
```

Comment:
Shouldn't it be a textarea instead?

Reply:
It's questionable. I took a input box because it's 1 line. I think @Rabuzarus has the final decision.

Reply:
I believe it makes for a better legibility if you can see the full schemestring.